src/languages/cpp.js

```diff
@@ -47,7 +47,7 @@ function(hljs) {
     keywords: {
       'meta-keyword':
         'if else elif endif define undef warning error line ' +
-        'pragma ifdef ifndef include'
+        'pragma ifdef ifndef include __has_include __has_cpp_attribute _Pragma'
```

Comment:
Please remove the `__keywords` (double underscore).  If we're going to do this we need to do it completely, not partially:

https://github.com/highlightjs/highlight.js/issues/2173

Reply:
Done.